README.md

```diff
@@ -1,3 +1,41 @@
-# harvester-prototype
 
-This repository is for the frontend of the project react
+# Harvester Prototype (FertiScan)
+
+## Overview Description
+
+---
+The Harvester project, represents a significant 
+advancement in fertilizer bag inspection. The primary goal of this prototype is
+to test and validate various potential solutions that could be integrated into 
+the final project. The AI-Lab team, utilizing unit tests, strives to determine 
+the best approach to address issues associated with fertilizer bag inspection.
```

Comment:
As an outsider of the project, I would like to know in the overview, what is the goal of the project and how can Harverster help the inspection? Right now your description is oriented too much on the development part which is good, but doesn't indicate what the project is about.

Think about the 5 w (Why, Who, What, Where, When)
- Why is the AI-Lab doing this
- Who is implicated in the project (Ai-Lab, CFIA and Client (inspecter, teams, etc.)
-  What is the project
-  Where (could be like web application)
- I think you can skip the When here

Reply:
@MaxenceGui  how do you find the new version?

Reply:
Way better 👍🏻